src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -228,28 +234,42 @@ export default function SnapshotListModal() {
               <Tbody>
                 {snapshotsList.map(
                   (
-                    { uuid: snap_uuid, created_at, content_counts, added_counts, removed_counts }: SnapshotItem,
+                    {
+                      uuid: snap_uuid,
+                      created_at,
+                      content_counts,
+                      added_counts,
+                      removed_counts,
+                    }: SnapshotItem,
                     index: number,
                   ) => (
                     <Tr key={created_at + index}>
                       <Td>{new Date(created_at).toUTCString()}</Td>
                       <Td>
-                        <ChangedArrows
-                          addedCount={
-                            (added_counts?.['rpm.advisory'] || 0) +
-                            (added_counts?.['rpm.package'] || 0)
-                          }
-                          removedCount={
-                            (removed_counts?.['rpm.advisory'] || 0) +
-                            (removed_counts?.['rpm.package'] || 0)
-                          }
-                        />
+                        {snap_uuid && (
+                          <Link to={`${rootPath}${SnapshotListModalRoute(snap_uuid)}`}>
```

Comment:
You are using a react-router link component here. 
Although this may work, I think as you've created functions above, you'll likely want to use a button to encapsulate the changed arrows with an [inline link](https://www.patternfly.org/components/button) button, as seen on the Patternfly component page. 
You can then use the regular "onClick" or the "href" variable depending.